.github/workflows/workflow_call.ruff.yml

```diff
@@ -31,4 +31,5 @@ jobs:
       - name: ruff
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OUTPUT_FORMAT: github
```

Comment:
`$OUTPUT_FORMAT` is set by the `Makefile` when running on GitHub actions (`$GITHUB_ACTIONS=true`). This is not necessary.